raisinbread/README.md

```diff
@@ -49,13 +58,14 @@ cat SQL/0000-00-00-schema.sql \
     raisinbread/RB_files/*.sql | mysql
 ```
 
-Note: to empty and delete all RaisinBread tables, use the following command in the 
-main LORIS root directory
-```bash
-cat raisinbread/instruments/instrument_sql/9999-99-99-drop_instrument_tables.sql \
-    SQL/9999-99-99-drop_tables.sql | mysql
+***Important:** Ensure that the above commands (including those to drop tables) were all completed properly.* 
+If there is a foreign key constraint error, run the following command on the mysql command line to mitigate the error:
+
+```bash 
+SET FOREIGN_KEY_CHECKS=0;
```

Comment:
We generally discourage the use of the `SET FOREIGN_KEY_CHECKS=0;` as it is generally a sign that there is an issue somewhere else basically. I will not block this addition in this PR because the ultimate purpose is to erase the database however the following points are very important.

- `SET FOREIGN_KEY_CHECKS=0;` can only be used right before the droping of the tables and should immediately be undone with `SET FOREIGN_KEY_CHECKS=1;` after the dropping and before the sourcing of the schema !!
- the need for `SET FOREIGN_KEY_CHECKS=0;` signifies that there is an issue with our `9999-99-99 ...` drop patches which should be corrected instead of bypassed. which means depending on the reason the drops are failing with foreign key issues, the developer should instead resolve the issue and submit a fix to LORIS

Reply:
I can remove this section and instead work on fixing the bug with the `9999-99-99 ... `patches. Is that a better approach?

Reply:
yes for sure